source/transaction_api.rst

```diff
@@ -35,11 +35,13 @@ service_charge              No          A number indicating service charge (in R
 tax                         No          A number indicating transaction tax (in Rp.)
 channel                     No          Channel of a transaction, for channel mapping, see table below
 type                        No          The type of prepared transactions, for type mapping, see table below
-items                       No          List of items containing product name, quantity, subtotal &
-                                        stamps_subtotal (optional).
+items                       No          List of items containing product name, quantity, subtotal,
+                                        stamps_subtotal (optional) & eligible_for_stamps (optional).
                                         ``price`` is the combined price of products (qty * unit price),
                                         ``stamps_subtotal`` is the combined stamps of products (qty * unit stamps),
                                         this field is optional.
+                                        ``eligible_for_stamps`` is boolean value to determine whether the item is considered in stamps calculation or not.
```

Comment:
`boolean value to determine whether the item should be included in stamps. Defaults to ``true```